api/web_service.py

```diff
@@ -34,3 +35,19 @@ def get_history(self):
 
     def thread_proc(self):
         self.app.run("localhost", 5000)
+
+    def create_bot(self):
+        try:
+            data = request.json
+            ema_short = int(data['ema_short'])
+            ema_long = int(data['ema_long'])
+            symbol = data['symbol']
+            if ema_short >= ema_long or symbol not in ["BTCUSD", "ETHUSD"] or ema_short > 500 or ema_long > 1000:
```

Comment:
We'll want this symbol list to be accessible to the UI as well so we can limit customers to these symbol names. It doesn't have to be in this PR though